src/copick_utils/segmentation/segmentation_from_picks.py

```diff
@@ -128,26 +76,41 @@ def segmentation_from_picks(radius, painting_segmentation_name, run, voxel_spaci
         seg = segs[0]
 
     segmentation_group = zarr.open_group(seg.path, mode="a")
-    
-    # Ensure that the tomogram Zarr is available
-    tomogram_group = zarr.open(tomogram.zarr(), "r")
-    multiscale_levels = list(tomogram_group.array_keys())
-
-    # Iterate through multiscale levels to paint picks as spheres
-    for level in multiscale_levels:
-        level_data = tomogram_group[level]
-        shape = level_data.shape
-
-        # Load or initialize the segmentation array in memory for the current level
-        if level in segmentation_group:
-            painting_seg_array = np.array(segmentation_group[level])
-        else:
-            painting_seg_array = np.zeros(shape, dtype=np.uint16)
+    highest_res_name = "0"
+
+    # Get the highest resolution dimensions and create a new array if necessary
+    highest_res_shape = segmentation_group[highest_res_name].shape if highest_res_name in segmentation_group else tomogram.shape
```

Comment:
`tomogram.shape` does not exist in the copick API, this raises an error for me. Should this simply raise an error if "0" is not found? 

The names of the multiscale levels should be pretty standard. I would always expect "0" there.

Reply:
Ah, just realized the purpose of this. This should probably be something like: 

```python
tomo_shape = zarr.open(tomogram.zarr(), "r")["0"].shape

# Get the highest resolution dimensions and create a new array if necessary
highest_res_shape = segmentation_group[highest_res_name].shape if highest_res_name in segmentation_group else tomo_shape
```